.gitignore

```diff
@@ -1,9 +1,31 @@
-NOTES.md
-release
-vendor
-*.sqlite
-*.sqlite3
-*.bak
-*.out
-*.db
-*.env
```

Comment:
any reason we need to change the .gitignore?  the newer version specifies a lot of files that will never exist in the project, but excludes files that should be ignored like sqlite files.

Reply:
Yeah the sqlite extensions are indeed missing now... The rest is just a copy from some standard go gitignore file.